tests/integration/tables/cpuid.cpp

```diff
@@ -31,7 +31,10 @@ TEST_F(cpuid, test_sanity) {
   };
 
   auto const data = execute_query("select * from cpuid");
+#if defined(__x86_64__)
+  // Not impltemented on AArch64
```

Comment:
Small typo

Reply:
Noted! I'll probably just not compile this test on aarch64 since that would be easier